core/src/main/kotlin/configuration.kt

```diff
@@ -50,18 +53,18 @@ enum class Platform(val key: String) {
 data class DokkaSourceSetID(
     val moduleName: String,
     val sourceSetName: String
-) : Serializable {
+) {
     override fun toString(): String {
         return "$moduleName/$sourceSetName"
     }
 }
 
 fun DokkaConfigurationImpl(json: String): DokkaConfigurationImpl {
-    return Gson().fromJson(json, DokkaConfigurationImpl::class.java)
+    return parseJson(json)
 }
```

Comment:
Maybe make this oneline: `fun DokkaConfigurationImpl(json: String) = parseJson(json)`

Reply:
Done ✅